edgecast/resources/cps/certificate.go

```diff
@@ -0,0 +1,557 @@
+							Description: "[ HttpLarge, HttpSmall, Adn ]",
+						},
+						"hex_url": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+				Description: "Returns a null value.",
+			},
+			"expiration_date": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Description: "Indicates the timestamp at which this certificate will expire. \n" +
+					"Syntax: \n" +
+					"{YYYY}-{MM}-{DD}T{hh}:{mm}:{ss}.{ffffff}Z \n" +
+					"If the Certificate Authority (CA) is still processing the certificate request, then this property returns the following timestamp: \n" +
+					"0001-01-01T00:00:00Z",
+			},
+			"last_modified": {
+				Type:     schema.TypeString,
+				Computed: true,
+				Description: "Indicates the timestamp at which this request for a certificate was last modified. \n" +
+					"Syntax: \n" +
+					"{YYYY}-{MM}-{DD}T{hh}:{mm}:{ss}.{ffffff}Z ",
+			},
+			"modified_by": {
+				Type:     schema.TypeSet,
+				Computed: true,
+
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"user_id": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"portal_type_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "[ Customer, Partner, Wholesaler, Uber, OpenCdn ]",
+						},
+						"identity_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"identity_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "[ User, Client ]",
+						},
+					},
+				},
+				Description: "Returns a null value.",
+			},
+			"request_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Returns Enterprise.",
+			},
+			"thumbprint": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Returns a null value.",
+			},
+			"workflow_error_message": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Returns a null value.",
+			},
+
+			"auto_renew": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Determines whether this certificate will automatically renew prior to its expiration date.",
+			},
+			"certificate_authority": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Set to DigiCert.",
+			},
+			"certificate_label": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Sets the certificate's name. Specify a name that solely consists of alphanumeric characters, underscores, and dashes.",
+			},
+			"dcv_method": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Determines the method through which your control over the domains associated with this certificate will be validated.",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Sets the certificate's description.",
+			},
+			"domains": {
```

Comment:
since the properties are listed as individual items in `main.tf`, I think it would be better for this to be `domain` instead of `domains`